### DISCUSSION TITLE
3D mesh not shown on mobile view

### DISCUSSION THREAD

**Discussion by moosavi222** [google/model-viewer#3653] [Q&A]
Hi there ,I am facing some issues while viewing the 3D model on mobile view.When the faces of the mesh are alot , the touch like dragging and moving the mesh doesnt work properly and shifts to the left and moves out of the frame.Any possible solutions for this problem?.Thank you for your time.

https://user-images.githubusercontent.com/43718040/181906074-7a1786e7-10fe-428c-835e-9d8c73f89734.mp4

**Comment by elalish** [comment]
I think that's a problem with your CSS rather than MV. It looks like you might have the beginnings of an image carousel? We can't really help without a link to your page where we can repro.

**Comment by moosavi222** [comment]
after debugging I have realized that this issues comes when I am using Vue.js. Cause its working fine without it.do you have  any solutions?
